src/pages/ReviewSession.tsx

```diff
@@ -69,7 +69,7 @@ const WrappingUpTxt = styled.h4`
   color: white;
 `;
 
-const animationDuration = 1.5;
+const animationDuration = 1.0;
```

Comment:
Would you mind creating a separate PR for this "Wrapping Up" animation duration? I know it's only a one-line change, but I generally try (keyword, "try" lol) to keep unrelated changes in separate PRs as it keeps things cleaner and is easier to revert if some changes end up causing a bug

Reply:
Sure thing!